addon/utils/d3-localization.js

```diff
@@ -26,3 +26,8 @@ export function getTickFormat({ periods = ['AM', 'PM'], dayFormat = '%a %d', wee
             : formatYear)(date);
     };
 }
+
+export function convertTZ(date, tzString) {
+    return new Date((typeof date === 'string' ? new Date(date) : date).toLocaleString('en-US', { timeZone: tzString }));
```

Comment:
So why are we always returning a date with an en-US representation here?

Reply:
Good question. This code uses toLocalString() to get an accurate zoned representation of the time, and then we convert it back to a Date object. toLocaleString() requires a locale argument, but as far as I can tell it does not matter if we always use en-US inside this function because we just convert it back to a date object, which will be in the browser's time zone (which is different from locale). I tried it with the UI set to couple of different locales and saw no adverse effects.